aws/table_aws_lambda_function.go

```diff
@@ -362,6 +362,17 @@ func getLambdaFunctionUrlConfig(ctx context.Context, d *plugin.QueryData, h *plu
 
 	functionName := functionName(h.Item)
 
+	commonColumnData, err := getCommonColumns(ctx, d, h)
+	if err != nil {
+		return nil, err
+	}
+
+	awsCommonData := commonColumnData.(*awsCommonColumnData)
+	if awsCommonData.Partition == "aws-us-gov"{
```

Comment:
Can you please add a comment above the `if` statement stating why we need this special handling? The comment should link to the AWS article mentioning limitations in GovCloud and the error we get (please see the original issue for the article link)